src/pyubx2/ubxreader.py

```diff
@@ -66,6 +66,7 @@ def __init__(self, datastream, **kwargs):
         :param bool scaling: (kwarg) 1 = apply scale factors, 0 = do not apply (1)
         :param bool labelmsm: (kwarg) whether to label RTCM3 MSM NSAT and NCELL attributes (1)
         :param int bufsize: (kwarg) socket recv buffer size (1024)
+        :param bool parsing: a boolean flag to determine if the stream needs to be parsed

```

Comment:
improve docstring and make consistent with others

Reply:
Done